components/list/list-item-mixin.js

```diff
@@ -147,7 +154,7 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin-bottom: -1px;
 			}
 			:host([_has-nested-list-add-button]:not([selection-disabled]):not([skeleton])[selected]) [slot="before-content"] {
-				border-bottom-color: #b6cbe8;
+				border-bottom-color: var(--d2l-list-item-border-color);
```

Comment:
Another way to do this if we don't want to run the risk of consumers noticing that the variable exists is to do something like:
```css
border-bottom-color: ${useNewStyles ? '#b6cbe8' : '#b6cbe8'};
```

Also makes it a bit easier to clean up the flag later since with the variable it might not be immediately obvious that the variable itself can go away with the flag.

Reply:
Forgot to mention that `useNewStyles` could be set at the top of the file in a `const` to the result of `getFlag()`.